Services/CO.CDP.Localization/StaticTextResource.resx

```diff
@@ -2057,8 +2057,18 @@
     <value>return to organisation details</value>
   </data>
   <data name="BuyerParentChildRelationship_ResultsPage_NoResults" xml:space="preserve">
-    <value>No organisations found matching your search criteria </value>
-  </data>
+  <value>
+    &lt;p class=&quot;govuk-body&quot;&gt;
+      &lt;b&gt;0&lt;/b&gt; organisations found.
```

Comment:
The b should be a span with govuk-!-font-weight-bold as the class

https://gds-way.digital.cabinet-office.gov.uk/manuals/programming-languages/html.html#text-emphasis